### DISCUSSION TITLE
打包成exe时运行exe时显示找不到config.yaml这个问题怎么搞。是用pyinstaller打包的

### DISCUSSION THREAD

**Discussion by tslxxcxy** [RapidAI/RapidOCRPDF#14] [Q&A]
![Snipaste_2024-04-07_16-44-26](https://github.com/RapidAI/RapidOCRPDF/assets/44352986/22373a03-d3e1-4dd1-92bb-6074e2b37b92)

**Comment by tslxxcxy** [comment]
FileNotFoundError: models\ch_PP-OCRv4_det_infer.onnx does not exists.前面报一个找不到config.yaml文件解决了。现在这个是一个相对路径怎么搞，也修改了config.yaml里面的model也不行，帮忙看看

**Comment by SWHL** [comment]
已经在[link](https://github.com/RapidAI/RapidOCR/discussions/164?converting=1)回复

**Reply by tslxxcxy** [reply]
已经收到，我也回复了
